From 4ea1e91417c3c7db9baf65d5fa23ffd8458e2695 Mon Sep 17 00:00:00 2001
From: codeworkx <daniel.hillenbrand@codeworkx.de>
Date: Fri, 14 Sep 2018 14:35:14 +0200
Subject: [PATCH 13/13] lineage: disable adb auth on eng build

We have to set ro.adb.secure to 0 on eng builds to get access.

Change-Id: Ie128f78c52c2f46b0a26d57eeed6a186aa7d9b67
---
 config/common.mk | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/config/common.mk b/config/common.mk
index 792fbe73..b7fc260e 100644
--- a/config/common.mk
+++ b/config/common.mk
@@ -26,7 +26,10 @@ ifneq ($(TARGET_BUILD_VARIANT),user)
 PRODUCT_SYSTEM_DEFAULT_PROPERTIES += persist.sys.dun.override=0
 endif
 
-ifneq ($(TARGET_BUILD_VARIANT),eng)
+ifeq ($(TARGET_BUILD_VARIANT),eng)
+# Disable ADB authentication
+PRODUCT_SYSTEM_DEFAULT_PROPERTIES += ro.adb.secure=0
+else
 # Enable ADB authentication
 PRODUCT_SYSTEM_DEFAULT_PROPERTIES += ro.adb.secure=1
 endif
-- 
2.17.1

